odoo/models.py

```diff
@@ -4725,6 +4725,7 @@ def copy_translations(self, new, excluded=()):
         if old.id in seen_map[old._name]:
             return
         seen_map[old._name].add(old.id)
+        installed_langs = set(code for code, _ in self.env['res.lang'].get_installed())
```

Comment:
You could simplify the condition below by including `en_US` in the set
```suggestion
        valid_langs = set(code for code, _ in self.env['res.lang'].get_installed()) | {'en_US'}
```
and the condition below would simply be `lang in valid_langs`.